homeassistant/components/http/ban.py

```diff
@@ -76,12 +98,38 @@ async def ban_middleware(
         _LOGGER.error("IP Ban middleware loaded but banned IPs not loaded")
         return await handler(request)
 
+    ip_address_ = ip_address(request.remote)  # type: ignore[arg-type]
```

Comment:
The intent of not creating the IP address object was to not pay the overhead if they don't have bans enabled